Documentation/ceph-cluster-crd.md

```diff
@@ -151,6 +153,20 @@ To change the defaults that the operator uses to determine the mon health and wh
 - `ROOK_MON_HEALTHCHECK_INTERVAL`: The frequency with which to check if mons are in quorum (default is 45 seconds)
 - `ROOK_MON_OUT_TIMEOUT`: The interval to wait before marking a mon as "out" and starting a new mon to replace it in the quorum (default is 600 seconds)
 
+### Mgr Settings
+
+You can use the cluster CR to enable or disable any manager module. This can be configured like so:
+
+```yaml
+mgr:
+  modules:
+  - name: <name of the module>
+    enabled: true
+```
+
+Some modules will have special configuration to ensure the module is fully functional after being enabled. Specifically:
+- `pg_autoscaler`: Rook will configure all new pools with PG autoscaling by setting: `osd_pool_default_pg_autoscale_mode = on`
```

Comment:
Should we be clear that this will be set on every orchestration?

Reply:
Everything in the cluster CR is effectively set on every orchestration. Is this different?

Reply:
Not really I guess.